docs/website/docs/dlt-ecosystem/llm-tooling/mcp-server.md

```diff
@@ -0,0 +1,202 @@
+
+
+:::note
+The MCP is progressively being adopted and not all clients support all the features used by the dlt MCP server. See [this page](https://modelcontextprotocol.io/clients) for an overview of client capabilities
+:::
+
+
+## Features
+
+The `dlt` and `dlt+` MCP servers aim to be a toolbox to help developers build, maintain, and operate `dlt` pipelines. There are two primary avenues:
+
+- **Code generation**: LLMs are good at writing Python code, but they don't know everything about `dlt`. The MCP provides resources and tools to provide up-to-date information to the LLm about the dlt library and the specifics of your project.
+
+- **Assistance**: The MCP resources, tools, and prompts can also be used to learn more about `dlt`. The built-in knowledgebase combined with your project's context brings Q&A support inside your IDE.
+
+The next sections are a non-exhaustive documentation of existing and upcoming features.
+
+### Tools
+
+- Use your local pipeline directory (default: `~/.dlt/pipelines`) to retrieve pipeline metadata. This includes: available pipleines, available tables, table schemas. It also include operational information such as last load date, schema changes, schema diffs, load packages data.
+
+- Use your pipeline context to inspect your dlt configuration (sources, resources, destinations).
+
+- Connect to the destination to retrieve data. This enables table previews and light text-to-SQL capabilities.
+
+### Resources
+
+- LLM-optimized dlt documentation pages. These can be selected and added to your prompt to help the LLM generate valid `dlt` code.
+
+### Prompts
+
+- Tutorial-like instructions that puts the LLM in "assistant mode". For example, the LLM can ask you questions about your data and "think" with you to select the right loading strategy.
+
+- Command-like instructions that gives the LLM a task to complete. For example, the LLM can initialize a new pipeline. This is akin to a conversational command line tool.
+
+
+## Installation
+
+The `dlt` and `dlt+` MCP servers are intended to run locally on your machine and communicate over standard I/O. Typically, the MCP server process is launched by the MCP client, i.e., the IDE. We will use the [uv package manager](https://docs.astral.sh/uv/#installation) to launch the MCP server.
+
+The next sections include client-specific instructions, references, and snippets to configure the MCP server. They are mainly derived from this `uv` command:
+
+```sh
+uv tool run --with "dlt-plus[mcp]==0.9.0" dlt mcp run
+```
+
+To explain each part:
+- [uv tool run](https://docs.astral.sh/uv/guides/tools/) executes the command in an isolated virtual environment
+- the `--with PACKAGE_NAME` specify the Python dependencies required by the MCP server
+- `dlt-plus==0.9.0` pins a specific `dlt-plus` version (where the MCP code lives). We suggest at least pinning the `dlt-plus` version to provide a consistent experience
+- `dlt mcp run` is a CLI command found in `dlt+`
```

Comment:
This one is missing the explanation of what it does, e.g., starts an MCP server.